src/database.js

```diff
@@ -54,6 +54,87 @@ function shutdownSQL() {
   }
 }
 
+/**
+ * @async
+ * @function insertNewPackage
+ * @desc Insert a new package inside the DB taking a `Server Object Full` as argument.
+ * @param {object} pack - The `Server Object Full` package.
+ * @returns {object} A Server Status Object.
+ */
+async function insertNewPackage(pack) {
+  try {
+    // TODO: Not sure here if we have to stringify all the package or
+    // only some properties of it.
+    const pack_data = JSON.stringify(pack);
```

Comment:
```suggestion
    const pack_data = pack;
```

Since the field `data` of `packages` on the database is a `JSON_B` object, we actually don't need to stringify the JSON at all, its compatible with zero changes. So we can just push this data into the column with no concerns. Good reference to how we migrated the data from JSON before is available on [`atom-package-migrator-to-db-2`](https://github.com/confused-Techie/atom-package-migrator-to-db-2/blob/main/main.js)